packages/apps/plugins/plugin-thread/src/components/ThreadContainer.tsx

```diff
@@ -97,7 +98,7 @@ export const ThreadContainer = ({
   const textboxMetadata = useMessageMetadata(thread.id, identity);
 
   return (
-    <Thread onClickCapture={onAttend} onFocusCapture={onAttend} current={current}>
+    <Thread onClickCapture={onAttend} onFocusCapture={onAttend} current={current} id={thread.id}>
```

Comment:
prefer our props up-front (esp. id) and callbacks at the end semi-standard.

Reply:
Will keep that in mind 👍